### PR TITLE
Lb/88 placement school details page for support users

### PR DESCRIPTION
**Pull request by elceebee**
## Context

Support users should be able to navigate from the list of organisations to the detail page for a school

## Changes proposed in this pull request
- Added lots of fields for the gias school
- A show page for the school to match the beta prototype designs

There are two things in the designs that are not included in the PR:
1.  The primary and secondary nav bars as those are being completed in other PRs
2. The links row actions "Change", as we don't have anywhere for those change links to go yet. 

## Guidance to review
- You'll need to run migrations and update the gias schools with the rake task (`rake gias_update`) to bring in all the new gias school fields

## Link to Trello card

https://trello.com/c/gyMTPatr

## Things to check

- [NA] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [x] This code does not rely on migrations in the same Pull Request (it does, but that's ok)
- [NA] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [NA] API release notes have been updated if necessary
- [NA] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [NA] Required environment variables have been updated or [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

## Screenshots

https://github.com/DFE-Digital/itt-mentor-services/assets/44073106/7e603dc2-c55a-4030-b003-a34570f0dcf5

### CODE REVIEW

**Comment by elceebee** [comment]
[Placements School details page for support users](https://trello.com/c/gyMTPatr/88-placements-school-details-page-for-support-users)

File: app/decorators/school_decorator.rb [hunk 1]
```diff
@@ -6,4 +6,8 @@ def formatted_address
     address_parts.reject!(&:blank?)
     address_parts.join("<br/>").html_safe
   end
+
+  def formatted_inspection_date
+    last_inspection_date&.strftime("%e %B %Y")
```
Comment (JamieCleare2525) [inline app/decorators/school_decorator.rb:11]: Is it worth moving this date format to the locales?

Comment (elceebee) [reply app/decorators/school_decorator.rb]: Resolved in [Add date and time formats to localisation](https://github.com/DFE-Digital/itt-mentor-services/pull/81/commits/dd75a7cc585f05b1af9c9f8664934cc0162c802b)

File: spec/system/placements/view_school_as_support_user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+RSpec.describe "Placements / Organisations / Support user views a School", type: :system do
+  let(:school) { create(:school, :placements) }
+
+  before do
+    school
+    Capybara.app_host = "http://#{ENV["PLACEMENTS_HOST"]}"
+  end
+
+  after { Capybara.app_host = nil }
```
Comment (JamieCleare2525) [inline spec/system/placements/view_school_as_support_user_spec.rb:11]: @Nitemaeric suggests that this is no longer needed.

**Review by JamieCleare2525 — APPROVED** [review]
LGTM! Tested Locally!

**Pull request merged**